lib/cdo/test_run_utils.rb

```diff
@@ -27,6 +27,13 @@ def self.run_apps_tests
     end
   end
 
+  def self.run_local_ui_test
+    feature_path = File.expand_path(ENV['feature'])
+    Dir.chdir(dashboard_dir('test/ui/')) do
+      RakeUtils.system "./runner.rb --verbose --dashboard=localhost-studio.code.org:3000 --local --feature=#{feature_path}"
```

Comment:
I believe some of our tests need to know the pegasus domain too; might be worth adding its default as well:
```
--pegasus=localhost.code.org:3000
```